src/main/resources/application.properties

```diff
@@ -31,3 +31,6 @@ spring.jpa.show-sql=true
 # schema will be automatically updated accordingly to java entities found in
 # the project
 spring.jpa.hibernate.ddl-auto=update
+
+# Naming strategy
+server.servlet.contextPath=/api
```

Comment:
did you put this line in your "-dev" file as well ?

Reply:
Yes!